tensorflow/training/docker/2.0/py3/cu100/Dockerfile.gpu

```diff
@@ -28,7 +28,7 @@ ARG PIP=pip3
 ARG OPENSSL_VERSION=1.1.1g
 
 ARG TF_URL=https://tensorflow-aws.s3-us-west-2.amazonaws.com/2.0.2/AmazonLinux/gpu/final/tensorflow_gpu-2.0.2-cp36-cp36m-manylinux2010_x86_64.whl
-ARG SMDEBUG_VERSION=0.9.2
+ARG SMDEBUG_VERSION=0.9.4
```

Comment:
Is `"sagemaker-tensorflow-training>=3.0<4.0"` version correct in this dockerfile, or should it be changed to `"sagemaker-tensorflow-training>=20"`?